db/migrate/20190320170708_create_approval_requests.rb

```diff
@@ -0,0 +1,12 @@
+class CreateApprovalRequests < ActiveRecord::Migration[5.2]
+  def change
+    create_table :approval_requests do |t|
+      t.string :approval_request_ref
+      t.string :workflow_ref
+      t.string :status
```

Comment:
Wondering if we should do `state` instead of `status` for consistency with Order?  @mkanoor do you have any thoughts on this?

Reply:
@syncrou I think state seems better, should we have a list of valid states somewhere in the model
[Processing, Approved, Denied] being the valid states

Reply:
Good catch @syncrou.

Do we know the valid states at this time?  If not we should be good to merge once the column name is changed and add the validation in a future PR.

Reply:
I only know of "approved" as the workflow I worked with was auto approval.  The initial kafka message payload sends back the request_id so I only saw `:request_id` & `approved`.  Though I believe the states are all defined explicitly in the approval-api.  @hsong-rh?

Reply:
https://github.com/ManageIQ/approval-api/blob/master/app/models/concerns/approval_decisions.rb

Looking here it does look like we can add make this class an enum of these stages

Reply:
Updated column name, we can handle the validation in the future (if needbe).